engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,7 +74,18 @@ public void resetCamera() {
     }
 
     @ReceiveEvent
-    public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef client, AutoMountCameraComponent autoMountCameraComponent, ClientComponent clientComponent) {
+    public void resetCamera(ResetCameraEvent resetCameraEvent, EntityRef entityRef, AutoMountCameraComponent autoMountCameraComponent, ClientComponent clientComponent) {
+        if (localPlayer.getClientEntity().equals(entityRef)) {
+            clientComponent.camera.destroy();
+            clientComponent.camera = EntityRef.NULL;
+            // this will trigger a ClientComponent change which will in turn trigger a remount
+            localPlayer.getClientEntity().saveComponent(clientComponent);
+        }
+    }
+
+    @Command(shortDescription = "Mounts the camera to the character", requiredPermission = PermissionManager.NO_PERMISSION)
```

Comment:
Yes, that's what I was wondering about. But perhaps even just as a debug tool it might be worth keeping it. Perhaps we could put an expiration date on it, i.e. removing it after a number of releases so that we can use it for testing purposes while you are working on cameras and some time after.